src/modules/metadata/actions.js

```diff
@@ -84,7 +84,7 @@ const _fetchExtraPropertiesSchemaTypes = networkAction(() => (dispatch, getState
     error: "Error fetching extra properties schema types",
 }));
 
-export const fetchExtraPropertiesSchemaTypes = () => (dispatch, getState) => {
+export const fetchExtraPropertiesSchemaTypes = (dispatch, getState) => {
```

Comment:
true that this doesn't need to be a double-function anymore, but it does make it a bit inconsistent with other actions in bento...  hmm